noir-projects/aztec-nr/aztec/src/discovery/private_logs.nr

```diff
@@ -54,8 +53,7 @@ pub unconstrained fn do_process_log<Env>(
     // currently just have two log types: 0 for private notes and 1 for partial notes. This will likely be expanded and
     // improved upon in the future to also handle events, etc.
 
-    let (storage_slot, note_type_id, log_type_id, log_payload) =
-        destructure_log_plaintext(log_plaintext);
+    let (log_type_id, log_metadata, log_content) = extract_log_type_id(log_plaintext);
```

Comment:
Why this naming change? Given that it extracts also metadata and content the func name is misleading.

Reply:
Yeah good point. I renamed it to `decode_log_plaintext` and added docs.